Orange/widgets/data/owtranspose.py

```diff
@@ -7,6 +11,28 @@
 from Orange.widgets.widget import Input, Output
 
 
+def add_radio_buttons(widget, master, value, btn_labels=(), orientation=Qt.Vertical, callback=None):
```

Comment:
I don't like this function at all. It is called `add_radio_buttons`, and it appends radio buttons and line edits. And then a combo is added separately since this function handles only line edits. (Well, if it handled everything, it would just replace any placing of components inside control area.)

It looks general, but is called only once and replace what we would otherwise do with a sequence of calls to `gui` method that we usually use for constructing the interface. I would suggest refactoring this code into `__init__`.

Reply:
I wanted to make it general, well then I moved everything into `__init__`.